### PR TITLE
[FIX] App isn't showing message for PDF/file uploads

### PR DESCRIPTION
**Pull request by saket-m0**
## Proposed changes

The app now shows the message entered with the pdf or other files on sending.

In app/containers/message/Reply.js added an if statement to check if file description exists and if yes then add a markdown with value msg equal to the description.

Also tested using 'yarn test -u' to add/update the tests.

## Issue(s)	

Fixing the issue #2531 
## Screenshots
The text entered while sending the file is visible in the development version now (Refer to the latest attachment sent).
Screenshot_20201010-000013_ DEBUG  RocketChat Experimental

The same text is not visible in the production version (Refer to the latest attachment sent).
Screenshot_20201009-234131_RocketChat

## Types of changes

- [ x ] Bugfix (non-breaking change which fixes an issue)
- [ x ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [ x ] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [ x ] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [ x ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

I am new to Opensource. Please let me know if I have done something wrong so that I can Fix It. Thank You!

### CODE REVIEW

**Comment by saket-m0** [comment]
Hello, I have added a pull request. The pdfs now show messages. 
I ran the tests locally with 
```sh
yarn test -u
```
and the tests were successful.

On the pull request it is showing 2 check have been failed. `iOS Detox / detox-build (pull_request)` and `ci/circleci: lint-testunit` have been failed and I dont know why. Can someone please help me.

**Comment by diegolmello** [comment]
> Hello, I have added a pull request. The pdfs now show messages.
> I ran the tests locally with
> 
> ```shell
> yarn test -u
> ```
> 
> and the tests were successful.
> 
> On the pull request it is showing 2 check have been failed. `iOS Detox / detox-build (pull_request)` and `ci/circleci: lint-testunit` have been failed and I dont know why. Can someone please help me.

Don't worry about it.
We're going to review it soon.
Thanks.

**Comment by saket-m0** [comment]
Oh ok. Thanks a lot!

File: app/containers/message/Reply.js [hunk 1]
```diff
@@ -146,6 +146,44 @@ const Reply = React.memo(({
 		openLink(url, theme);
 	};
 
+	if (attachment.desctiption) {
```
Comment (djorkaeffalexandre) [inline app/containers/message/Reply.js:149]: typo

File: app/containers/message/Reply.js [hunk 2]
```diff
@@ -146,6 +146,44 @@ const Reply = React.memo(({
...
+					<Fields attachment={attachment} theme={theme} />
+				</View>
+				<Markdown
+					msg={attachment.description}
+					baseUrl={baseUrl}
+					username={user.username}
+					getCustomEmoji={getCustomEmoji}
+					theme={theme}
+				/>
+			</Touchable>
```
Comment (djorkaeffalexandre) [inline app/containers/message/Reply.js:183]: You can just add Markdown component to Touchable Component, line 187.
`<Touchable>` should receive only one single child, so, use `<>` to wrap `attachmentContainer` and description.

**Review by djorkaeffalexandre — CHANGES_REQUESTED** [review]

**Pull request closed without merging**